packages/internal/src/__tests__/nestedPages.test.ts

```diff
@@ -183,20 +235,28 @@ describe('User specified imports, with static imports', () => {
 
     expect(outputWithStaticImports).toContain('var _EditJobPage = require("')
 
-    expect(outputWithStaticImports).toContain(`const EditJobPage = {
-  name: "EditJobPage",
-  prerenderLoader: name => require("./pages/Jobs/EditJobPage/EditJobPage"),
-  LazyComponent: (0, _react.lazy)(() => import( /* webpackChunkName: "EditJobPage" */"./pages/Jobs/EditJobPage/EditJobPage"))
-}`)
+    expect(outputWithStaticImports).toContain(
+      normalizeStr(
+        `const EditJobPage = {
+          name: "EditJobPage",
+          prerenderLoader: name => require("./pages/Jobs/EditJobPage/EditJobPage"),
+          LazyComponent: (0, _react.lazy)(() => import( /* webpackChunkName: "EditJobPage" */"./pages/Jobs/EditJobPage/EditJobPage"))
+        }`
+      )
+    )
 
     expect(outputNoStaticImports).toContain(
       'var _EditJobPage = _interopRequireWildcard('
     )
 
-    expect(outputNoStaticImports).toContain(`.createElement(_router.Route, {
-    path: "/jobs/{id:Int}/edit",
-    page: _EditJobPage["default"],
-    name: "editJob"`)
+    expect(outputNoStaticImports).toContain(
+      normalizeStr(
+        `}), /*#__PURE__*/(0, _jsxRuntime.jsx)(_router.Route, {
```

Comment:
The main difference in these tests is that they look for `/*#__PURE__*/(0, _jsxRuntime.jsx)` instead of `.createElement`